src/js/grapl-cdk/lib/grapl-cdk-stack.ts

```diff
@@ -392,6 +399,46 @@ class EngagementCreator extends cdk.NestedStack {
     }
 }
 
+interface DGraphSwarmClusterProps {
+    prefix: string;
+    vpc: ec2.IVpc;
+}
+
+export class DGraphSwarmCluster extends cdk.NestedStack {
+    private readonly dgraphAlphaZone: route53.PrivateHostedZone;
+    private readonly dgraphSwarmCluster: Swarm;
+
+    constructor(
+        parent: cdk.Construct, id: string, props: DGraphSwarmClusterProps
+    ) {
+        super(parent, id);
+
+        this.dgraphAlphaZone = new route53.PrivateHostedZone(
+            this,
+            id + "-DGraphSwarmZone",
```

Comment:
For `id`s like this I suggest we don't concat the parent `id`. Just pass `DGraphSwarmZone` and let CDK do the rest for us. We used to have to do this for classes that didn't derive from cdk.Construct (indirectly here), but when it does we should avoid it.

Reply:
For posterity, Ian and I talked in Slack and this will be addressed in a subsequent PR.